### PR TITLE
Fix Issue #398

### LINKED ISSUE #398: Close the presence palette when the activity is shared
When an user want to share an activity, he click on the presence palette, then on the neighborhood icon. 

image

In some activity, when the icon is clicked, the palette automatically close. For example in Maze activity. On some other, the palette remain open which is not pertinent.

The idea is to generalize this feature by calling `popDown` method like [here](https://github.com/llaske/sugarizer/blob/master/activities/MazeWeb.activity/js/activity.js#L886).

So the following activities should be fixed:

* [ ] Chat activity
* [ ] Memorize activity
* [ ] Paint activity

### PR DESCRIPTION
**Pull request by AndreaGon**
Added presencepalette.popdown() to close presence palette when activity is shared. Hopefully fixes #398.

- [x] Chat Activity

- [x] Memorize Activity

- [x] Paint Activity

### CODE REVIEW

File: activities/ChatPrototype.activity/js/activity.js [hunk 1]
```diff
@@ -45,7 +45,9 @@ define(["sugar-web/activity/activity","webL10n","sugar-web/graphics/palette","su
 		// Connect to network
         var presenceObject;
 		function shareActivity() {
+      			presencepalette.popDown();
```
Comment (FreddieN) [inline activities/ChatPrototype.activity/js/activity.js]: You have changed the indentation of this line compared to the file. Could you make sure that it is all in line?

File: activities/Memorize.activity/js/activity.js [hunk 2]
```diff
@@ -81,6 +81,7 @@ function initPresence(activity, memorizeApp, presencepalette, callback) {
             presencePalette.setShared(true);
         } else {
             presencePalette.addEventListener('shared', function () {
+              presencePalette.popDown();
```
Comment (FreddieN) [inline activities/Memorize.activity/js/activity.js]: This needs to move over similar to above so it is in line with shareActivity.

**Comment by AndreaGon** [comment]
@FreddieN Is this how it should be?

**Comment by llaske** [comment]
Good job. Thanks.

**Pull request merged**